src/pretix/api/views/webhooks.py

```diff
@@ -19,18 +19,26 @@
 # You should have received a copy of the GNU Affero General Public License along with this program.  If not, see
 # <https://www.gnu.org/licenses/>.
 #
+import django_filters
+from django_filters.rest_framework import FilterSet, DjangoFilterBackend
 from rest_framework import viewsets
 
 from pretix.api.models import WebHook
 from pretix.api.serializers.webhooks import WebHookSerializer
 from pretix.helpers.dicts import merge_dicts
 
 
+class EventFilter(FilterSet):
```

Comment:
Shouldn't it be more named like the corresponding ViewSet? :)

```suggestion
class WebHookFilter(FilterSet):
```